doc/logos/README.md

```diff
@@ -0,0 +1,50 @@
+# scikit-learn Branding
+
+This section contains scikit-learn's branding standards.
+File types:
+- `AI` is the filename extension used by Adobe Illustrator
```

Comment:
We do not provide logos in `AI`, `EPS` or `JPG`, so I do not think we need to state them?

The `SVG` file should be enough to be used in any editing software and can be scaled to any resolution (because it the SVG is a vector representation).

Reply:
OK, removed.